src/pages/search-page/results-dataset/dataset-item/dataset-item.component.tsx

```diff
@@ -45,12 +45,19 @@ const renderAccessRights = (accessRight: any) => {
 const renderThemes = (themes: any, losItems: any) => {
   return themes
     .map((theme: any, index: number) => {
-      const losItem = find(losItems, { uri: theme.id });
-      const title = getTranslateText(losItem?.prefLabel || theme.title);
+      const losItem: any =
+        Object.values(losItems).find(
+          (losItem: any) => losItem.uri === theme.id
+        ) || {};
+      const losPath: any = losItem?.losPaths ? losItem.losPaths[0] : '';
+      const prefLabel = getTranslateText(losItem.prefLabel);
       return (
-        title && (
-          <RoundedTag key={`dataset-theme-${index}`} to="">
-            <span>{title}</span>
+        prefLabel && (
+          <RoundedTag
+            key={`dataset-theme-${index}`}
+            to={`${PATHNAME_DATASETS}${patchSearchQuery('losTheme', losPath)}`}
```

Comment:
Funker denne endringen for alle entiteter (dataset, api, begreper og info modeller) og ikke bare datasett?

Reply:
Dette er bare på dataset-item.component.tsx, altså bare for datasett.

Reply:
Dette skal i så fall funke på informasjonsmodeller også

Reply:
Yes, lagt til for informasjonsmodeller også.